src/dymaptic.GeoBlazor.Core/Components/Views/MapView.razor.cs

```diff
@@ -2071,8 +2082,8 @@ protected override async Task RenderView(bool forceRender = false)
         }
 
         Rendering = true;
-        Map.Layers.RemoveWhere(l => l.Imported);
-        Map.Basemap?.Layers.RemoveWhere(l => l.Imported);
+        Map.Layers.RemoveAll(l => l.Imported);
```

Comment:
This required a lookup on my part and makes sense combined with the other transitions from the use of `Hashset<T>` (https://learn.microsoft.com/en-us/dotnet/api/system.collections.generic.hashset-1.removewhere?view=net-8.0) to `List<T>`(https://learn.microsoft.com/en-us/dotnet/api/system.collections.generic.list-1.removeall?view=net-8.0)

Reply:
Yeah, this was just necessary due to the fact that they don't have the same remove methods.